mscore/inspector/inspector.cpp

```diff
@@ -1038,24 +1038,34 @@ InspectorTremolo::InspectorTremolo(QWidget* parent)
 void InspectorTremolo::setElement()
       {
       InspectorElementBase::setElement();
-      bool hasNonMinimTwoNoteTremolo = false;
-      bool hasTabStaffTremolo = false;
+
+      bool hasMinimTwoNoteTremoloOnNonTabStaves = false;
       for (Element* ee : *(inspector->el())) {
-            if (toTremolo(ee)->durationType().type() != TDuration::DurationType::V_HALF) {
-                  hasNonMinimTwoNoteTremolo = true;
-                  break;
-                  }
-            if (toTremolo(ee)->staffType()->group() == StaffGroup::TAB) {
-                  hasTabStaffTremolo = true;
+            if (  toTremolo(ee)->twoNotes()
+               && toTremolo(ee)->durationType().type() == TDuration::DurationType::V_HALF
+               && toTremolo(ee)->staffType()->group() != StaffGroup::TAB) {
+                  hasMinimTwoNoteTremoloOnNonTabStaves = true;
                   break;
                   }
             }
-      // beam style setting is only appliable to minim two-note tremolo in non-TAB staves
-      if (hasNonMinimTwoNoteTremolo || hasTabStaffTremolo) {
-            g.labelBeamStyle->setVisible(false);
-            g.beamStyle->setVisible(false);
-            g.resetBeamStyle->setVisible(false);
+      // beam style setting is only applicable to minim two-notes tremolos in non-TAB staves
+      // one of those in the selection is enough reason to show that part of the dialog
+      g.labelBeamStyle->setVisible(hasMinimTwoNoteTremoloOnNonTabStaves);
+      g.beamStyle->setVisible(hasMinimTwoNoteTremoloOnNonTabStaves);
+      g.resetBeamStyle->setVisible(hasMinimTwoNoteTremoloOnNonTabStaves);
```

Comment:
I still disagree ;-)
If the behaviour is similar elsewhere, then it should be changed too. Selecting a single tremolo which doesn't support this won't show this, but selecting it with another that does support this will show this: users will then expect the beam style of the tremolo which actually doesn't support this be changed to by this "trick". But turns out they can't. Which can be confusing.

Reply:
Just like a selection of staff type change elements: if any of them is attached on a tab staff, the "Show key signature" option should be disabled, because it doesn't make sense to a tab staff.

Reply:
If a property applies to one of the selected elements, it needs to get shown.
This is how it should be and how it is elsewhere (and in places where it is not that is a bug)

Reply:
I think otherwise.

Reply:
then rethink ;-)

Reply:
> Just like a selection of staff type change elements: if any of them is attached on a tab staff, the "Show key signature" option should be disabled, because it doesn't make sense to a tab staff.

Is it? Or should it? If it it is that'd be a bug in my book.

Reply:
This may lead to other problems. If the option is enabled even if there's non-minim or non-two-note tremolos, those tremolos will have the property recorded in files. And for them, the properties are useless. I didn't disable it in `libmscore` code, only in `mscore`.

Reply:
> Is it? Or should it (in your opinion)? If it is, that'd be a bug in my book.

It is. I had several conversations about this in #5535.

Reply:
Then this is a bug to fix

Reply:
> This may lead to other problems. If the option is enabled even if there's non-minim or non-two-note tremolos, those tremolos will have the property recorded in files. And for them, the properties are useless. I didn't disable it in `libmscore` code, only in `mscore`.

Then this is a bug to fix

Reply:
> > Is it? Or should it (in your opinion)? If it is, that'd be a bug in my book.
> 
> It is. I had several conversations with @dmitrio95 about this in #5535.

This: https://github.com/musescore/MuseScore/pull/5535#pullrequestreview-336447024 ?

Reply:
> Then this is a bug to fix. Applying properties to elements that don't have those is just plain wrong and a coding error

I agree to that, but I see no reason to agree to enabling the option to elements that don't need it.

Reply:
If there is no element that needs it, I certainly agree to not show it, this is exactly what this PR is about.
But if there is one or more elements in the selection that needs it, it needs to get shown.

Reply:
Again, I think otherwise ;-)
Let's ask for others' opinions then.